x-pack/plugins/security_solution/public/flyout/right/components/share_button.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { copyToClipboard, EuiButtonEmpty, EuiCopy } from '@elastic/eui';
+import type { FC } from 'react';
+import React from 'react';
+import { FLYOUT_URL_PARAM } from '../../url/use_sync_flyout_state_with_url';
+import { FLYOUT_HEADER_SHARE_BUTTON_TEST_ID } from './test_ids';
+import { SHARE } from './translations';
+
+interface ShareButtonProps {
+  alertUrl: string;
+}
+
+/**
+ * Puts alertUrl to users's clipboard. If current query string contains synced flyout state,
```

Comment:
typo I think? it would be either `users'` or `user's`

Reply:
fixed